markets/bfp-market/contracts/modules/MarketConfigurationModule.sol

```diff
@@ -49,10 +50,14 @@ contract MarketConfigurationModule is IMarketConfigurationModule {
         OwnableStorage.onlyOwner();
 
         PerpMarketConfiguration.Data storage config = PerpMarketConfiguration.load(marketId);
-
+        PerpMarketConfiguration.GlobalData storage globalConfig = PerpMarketConfiguration.load();
```

Comment:
Minor: You can pull this below PerpMarket.exists(marketId), just above the if check where its used.